lib/searchkick/bulk_indexer.rb

```diff
@@ -56,27 +56,24 @@ def batches_left
     end
 
     private
-
     def import_or_update(records, method_name, async)
-      if records.any?
-        if async
-          Searchkick::BulkReindexJob.perform_later(
-            class_name: records.first.class.name,
-            record_ids: records.map(&:id),
-            index_name: index.name,
-            method_name: method_name ? method_name.to_s : nil
-          )
-        else
-          records = records.select(&:should_index?)
-          if records.any?
-            with_retries do
-              # call out to index for ActiveSupport notifications
-              if method_name
-                index.bulk_update(records, method_name)
-              else
-                index.bulk_index(records)
-              end
-            end
+      return if records.none?
+      if async
```

Comment:
Please keep change set to a minimum (aka don't unnecessarily change indentation)